lib/mapObjects/CRewardableObject.cpp

```diff
@@ -317,16 +317,11 @@ bool CRewardableObject::wasVisited(PlayerColor player) const
 		case VISIT_UNLIMITED:
 		case VISIT_BONUS:
 			return false;
-		case VISIT_ONCE: // FIXME: hide this info deeper and return same as player?
```

Comment:
This might fix the Corpse for now, but the code was meant for configurable object support (unfinished). Please leave it commented out at least.